quodlibet/po/fr.po

```diff
@@ -2851,236 +2861,227 @@ msgstr ""
+#: ../quodlibet/ext/songsmenu/albumart.py:804
 #: ../quodlibet/ext/songsmenu/fingerprint/search.py:42
 msgid "Done"
-msgstr ""
+msgstr "Fait."
 
-#: ../quodlibet/ext/songsmenu/albumart.py:925
-#, fuzzy
+#: ../quodlibet/ext/songsmenu/albumart.py:922
 msgid "Download Album Art"
-msgstr "Jaquettes d'Album"
+msgstr "Télécharger des illustrations"
 
-#: ../quodlibet/ext/songsmenu/albumart.py:926
+#: ../quodlibet/ext/songsmenu/albumart.py:923
 msgid "Downloads album covers from various websites."
-msgstr ""
+msgstr "Téléchargement de jaquettes d'album depuis plusieurs sites Web."
 
-#: ../quodlibet/ext/songsmenu/albumart.py:939
+#: ../quodlibet/ext/songsmenu/albumart.py:936
 msgid "Sources"
-msgstr ""
+msgstr "Sources"
 
 #: ../quodlibet/ext/songsmenu/ape2id3.py:26
 msgid "APEv2 to ID3v2"
-msgstr ""
+msgstr "APEv2 vers ID3v2"
 
 #: ../quodlibet/ext/songsmenu/ape2id3.py:27
 msgid ""
 "Converts your APEv2 tags to ID3v2 tags. This will delete the APEv2 tags "
 "after conversion."
 msgstr ""
+"Convertit des étiquettes APEv2 en étiquettes ID3v2. Les étiquettes APEv2 "
+"sont supprimées après conversion."
 
 #: ../quodlibet/ext/songsmenu/bookmarks.py:23
-#, fuzzy
 msgid "Go to Bookmark"
-msgstr "_Modifier les signets"
+msgstr "Aller aux signets"
 
 #: ../quodlibet/ext/songsmenu/bookmarks.py:24
 msgid "Manages bookmarks in the selected files."
-msgstr ""
+msgstr "Gère les signets des fichiers sélectionnés."
 
 #: ../quodlibet/ext/songsmenu/bookmarks.py:63
 #: ../quodlibet/qltk/seekbutton.py:244
-#, fuzzy
 msgid "_Edit Bookmarks…"
-msgstr "_Modifier les signets"
+msgstr "Modifi_er les signets"
 
 #: ../quodlibet/ext/songsmenu/bookmarks.py:73
-#, fuzzy
 msgid "No Bookmarks"
-msgstr "Signets"
+msgstr "Aucun signet"
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:24
-#, fuzzy
 msgid "MusicBrainz Lookup"
-msgstr "Type MusicBrainz de l'album"
+msgstr "Recherche MusicBrainz"
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:26
 msgid "Re-tags an album based on a MusicBrainz search."
-msgstr ""
+msgstr "Étiqueter l'album à nouveau, en utilisant MusicBrainz."
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:50
 msgid "Only use year for \"date\" tag"
-msgstr ""
+msgstr "Indiquer uniquement l'année dans l'étiquette \"date\""
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:51
 msgid "Write \"_albumartist\" when needed"
 msgstr ""
+"Utiliser l'étiquette \"_albumartist\" (artiste auteur de l'album) si "
+"nécessaire"
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:52
 msgid "Write sort tags for artist names"
-msgstr ""
+msgstr "Trier les noms des artistes avant de les enregistrer dans l'étiquette"
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:53
 msgid "Write _standard MusicBrainz tags"
-msgstr ""
+msgstr "Écrire les étiquettes standard MusicBrainz"
 
 #: ../quodlibet/ext/songsmenu/brainz/__init__.py:54
 msgid "Write \"labelid\" tag"
-msgstr ""
+msgstr "Ecrire l'étiquette \"labelid\" (référence de l'étiquette)"
```

Comment:
Not sure what you mean by "(référence de l'étiquette)". labelid is MusicBrainz Identifier so just "Écrire l'étiquette \"labelid\"", MB users will understand, or "Écrire l'étiquette \"labelid\" (étiquette d'identification de MusicBrainz)"